common/lib/xmodule/xmodule/modulestore/inheritance.py

```diff
@@ -233,6 +234,33 @@ class InheritanceMixin(XBlockMixin):
         scope=Scope.settings
     )
 
+    @property
+    def close_date(self):
+        """
+        Return the date submissions should be closed from.
```

Comment:
Can you update the docstring and discuss how/why and on what conditions `grace period` is taking into account?

Reply:
Sure.